docs/ignoring-findings.md

```diff
@@ -4,47 +4,47 @@
 
 # Ignoring findings via inline comments
 
-Semgrep allows for ignoring, or whitelisting, findings in code by specifying a `nosem` comment on the first line of a finding. Comments take the form of `nosem` or `nosem: <rule-id>`. This functionality works across languages.
+Semgrep allows for ignoring findings in code by specifying a `nosemgrep` comment on the first line of a finding. Comments take the form of `nosemgrep` or `nosemgrep: <rule-id>`. This functionality works across languages.
```

Comment:
Thoughts on including a reference to `nosem` as well since it does work and historical findings may already have the comment applied?

Reply:
Yeah good point. I’ll try to keep it short.